lib/rules/web/admin_console/4.5/browser_opts.yml

```diff
@@ -21,6 +21,16 @@
             scroll_by = browser.element(css: ".navbar.navbar-pf-vertical").height
             scroll_by += browser.element(css: ".project-bar").height
             browser.execute_script("window.scrollBy(0,-#{scroll_by})")
+          when /pf-c-form__actions/
+            logger.info "Scrolling from action button out of view"
+            scroll_by = browser.element(css: ".pf-c-form__actions").height
+            scroll_by += browser.element(css: ".ocs-radio-group-field").height
+            browser.execute_script("window.scrollBy(0,-#{scroll_by})")
+          when /ocs-form-footer/
+            logger.info "Scrolling footer out of view"
+            scroll_by = browser.element(css: ".ocs-form-footer").height
+            scroll_by += browser.element(css: ".ocs-radio-group-field").height
+            browser.execute_script("window.scrollBy(0,-#{scroll_by})")
```

Comment:
If this is a footer, shouldn't we scroll down instead of scroll up? Perhaps remove the `-` sign. 

btw do we still need `nav-pf-secondary-nav` and `navbar-pf-vertical` scrolling?

Reply:
Another thing is that the offending element is `ocs-form-footer ocs-form-footer__sticky co-m-btn-bar`. So we only need to scroll by it only. No need to scroll by `.ocs-radio-group-field`.